test/integrationtest/projecttest/graphql/schema/flamingo.me_flamingo-commerce_v3_cart_interfaces_graphql-Service.graphql

```diff
@@ -363,6 +370,89 @@ input Commerce_BillingAddressFormInput {
     email:                  String!
 }
 
+input Commerce_DeliveryAddressFormInput {
+    vat:                    String
+    firstname:              String!
+    lastname:               String!
+    middleName:             String
+    title:                  String
+    salutation:             String
+    street:                 String
+    streetNr:               String
+    addressLine1:           String
+    addressLine2:           String
+    company:                String
+    city:                   String
+    postCode:               String
+    state:                  String
+    regionCode:             String
+    country:                String
+    countryCode:            String
+    phoneNumber:            String
+    email:                  String!
+}
+
+input Commerce_Cart_DeliveryAddressInput {
+    "Unique delivery code to identify the delivery"
+    deliveryCode: String!
+    "Delivery address form data"
+    deliveryAddress: Commerce_DeliveryAddressFormInput
+    "Should the data of the billing address be used for this delivery"
+    useBillingAddress: Boolean!
+    "Optional Shipping Method"
+    method: String
+    "Optional Shipping Method"
+    carrier: String
+}
+
+type Commerce_Cart_DeliveryAddressForm {
+    "Unique delivery code to identify the delivery"
+    deliveryCode: String!
+    "Delivery address form data"
+    formData:       Commerce_Cart_DeliveryAddressFormData
+    "Shows if the data of the billing address should be used for this delivery"
+    useBillingAddress: Boolean!
+    "Shipping Method"
+    method: String
+    "Shipping Carrier"
+    carrier: String
+    "Validation of supplied delivery address, empty if address is valid"
+    validationInfo: Commerce_Cart_Form_ValidationInfo
+    "Shows if the request was successfully processed"
+    processed: Boolean
+}
+
+type Commerce_Cart_DeliveryAddressFormData {
```

Comment:
Seems to be equal to Commerce_BillingAddressFormInput

1) Is it possible to use the same value object (e.g. a Commerce_Cart_Address)
2) when do we cleanup the missing namespace in existing objecte (Commerce_BillingAddressFormInput instead Commerce_Cart_BillingAddressFormInput)
3) here it is called "...AddressFormData"  in the other case "...BillingAddressFormInput" => Can we update the naming conventions (nt sure if we have a place where we specified the graphql naming conventions already?)

Reply:
Jeah I saw that too but wanted to keep these changes nonbreaking. But since the whole GraphQL part is still WIP I'll try to clean that up.

So far the [naming convention](https://github.com/i-love-flamingo/graphql#suggested-naming-conventions) only states: 
> We recommend to namespace your Types and Type extensions with the Project name. For Flamingo Core Framework GraphQL Schema we use the prefix Core_ and for Flamingo Commerce we use Commerce_

Reply:
- [x] Add missing `_Cart_` Namespace
- [x] Create one input/type for AddressForm and reuse it (`Commerce_Cart_AddressForm` / `Commerce_Cart_AddressFormInput`)

@danielpoe all done if you want to have a second look please go ahead. Will merge tomorrow otherwise.

Reply:
Cool +1